content-src/lib/first-run-data.js

```diff
@@ -0,0 +1,90 @@
+const IMAGE_PATH = "img/";
+const FIRST_RUN_TYPE = "first-run";
+const FAVICON_PATH = "favicons/images/";
+
+module.exports = {
+  FIRST_RUN_TYPE,
+  TopSites: [
```

Comment:
Random question, but any chance we can scrape this info from tippy-top-sites so things stay in sync?